pkg/authenticator/authenticator_test.go

```diff
@@ -27,26 +27,35 @@ func parseCert(filename string) (*x509.Certificate, error) {
 
 func TestAuthenticator(t *testing.T) {
 	Convey("IsCertExpired", t, func() {
-		Convey("Returns false if cert is not expired", func() {
+
+		Convey("Given a non-expired certificate to authenticate with Conjur", func() {
 			goodCert, err := parseCert("testdata/good_cert.crt")
-			So(err, ShouldBeNil)
 
-			authn := Authenticator{
-				PublicCert: goodCert,
-			}
+			Convey("Finishes without raising an error and returns certificate", func() {
+				So(err, ShouldBeNil)
+			})
 
-			So(authn.IsCertExpired(), ShouldEqual, false)
+			Convey("Returns the false that the certificate is not expired", func() {
```

Comment:
i think that "Returns false" is enough

Reply:
I am aiming for readability. Returning false doesn't really explain that false = not expired

Reply:
yes but when you read the whole context you see:
```
Given a non-expired certificate to authenticate with Conjur
    Returns false
```

i think this is better than:
```
Given a non-expired certificate to authenticate with Conjur
    Returns the false that the certificate is not expired
```

also, it is written in bad english :)

Reply:
it is clear that `false == not expired` because you write that this is your expectation